.gitattributes

```diff
@@ -0,0 +1,18 @@
+# This file is copied and modified from https://help.github.com/articles/dealing-with-line-endings/
+
+# Set the default behavior
+* text eol=lf
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
+*.txt text
+*.yml text
+
+# Denote all files that are truly binary and should not be modified.
+*.Rds binary
+*.rds binary
+*.png binary
+*.jpg binary
+*.feather binary
```

Comment:
i don't honestly know if this file helps at all. and I suspect it's not doing what I want....but maybe? seems worth a shot. goal is to make it so hashes of text files look the same on anyone's computer, i.e., so that EOLs are automatically all the same. decided on POSIX (Unix) for no particular reason except that it might be easier to see whether our computers are following the specification or not.